### ISSUE TITLE
/aux post bid on not working

### ISSUE THREAD

**Issue by Zoobug** [shirsig/aux-addon#379]
Playing WoW Classic on anniversary realms. 
The command /aux post bid remains off even when value is set to on

**Comment by Vanillabro** [comment]
Yes, I am having this issue as well.  This feature is one of the main reasons I love Aux.  Please fix this :(
